app/database/operator/server_data_operator/transformers/post.ts

```diff
@@ -112,6 +112,7 @@ export const transformDraftRecord = ({action, database, value}: TransformerArgs)
         draft.channelId = raw?.channel_id ?? '';
         draft.files = raw?.files ?? emptyFileInfo;
         draft.metadata = raw?.metadata ?? emptyPostMetadata;
+        draft.updateAt = raw.update_at || Date.now();
```

Comment:
```suggestion
        draft.updateAt = raw.update_at ?? Date.now();
```

I guess 0 is a valid value

Reply:
Make sense.